apps/bam.iobio/src/App.tsx

```diff
@@ -19,10 +19,12 @@
  *
  */
 
-import { createHistogram, createPercentBox } from '@overture-stack/iobio-components/components';
+import IobioComponents from '@overture-stack/iobio-components/components/src/index';
+import clsx from 'clsx';
 import { useEffect, useState } from 'react';
 import './App.css';
 
+const { Histogram, createPercentBox } = IobioComponents;
```

Comment:
Unclear why destructuring won't work in import

Reply:
we can look at it in a separate PR. this one has grown quite large already